### PR TITLE
[TASK] Add .travis.yml for the 1.2 branch

### PR DESCRIPTION
**Pull request by radmiraal**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      env: DB=sqlite
+    - php: 5.6
+      env: DB=mysql
+#    - php: 5.6
+#      env: DB=pgsql
+    - php: 5.5
+      env: DB=sqlite
+before_install:
+  - cd ..
+  - git clone https://github.com/neos/neos-base-distribution.git
```
Comment (hlubek) [inline .travis.yml]: I think we need to checkout the 1.2 branch of that repository here

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - cd neos-base-distribution
+install:
+  - composer require "neos/neos-development-collection dev-master" "neos/flow-development-collection dev-master"
```
Comment (hlubek) [inline .travis.yml]: We need to require the 1.2-dev branch here

**Comment by radmiraal** [comment]
:+1: travis now builds the 1.2 branch successfully

**Comment by hlubek** [comment]
:+1:

**Comment by aertmann** [comment]
:+1:

**Pull request merged**